lib/middleware/request_tracker.rb

```diff
@@ -155,12 +157,20 @@ def log_request_info(env, result, info)
 
   def call(env)
     result = nil
+    log_request = true
+    request = Rack::Request.new(env)
 
-    if rate_limit(env)
+    if rate_limit(request)
       result = [429, {}, ["Slow down, too Many Requests from this IP Address"]]
       return result
     end
 
+    if block_crawler(request)
+      log_request = false
+      result = [403, {}, []]
```

Comment:
we probably want some sort of message explaining this here

Reply:
👍